### PR TITLE
update to version 2.1.0-alpha-pre: don't use 2 numbers version

### PR DESCRIPTION
**Pull request by sblaisot**
short: see http://sourceforge.net/p/mixxx/mailman/mixxx-devel/thread/568EBACB.6020102%40mixxx.org/#msg34747625

long:
2.1-alpha-pre produces a windows MSI installer with version 2.1.gitrev (gitrev is the git revision number)
when 2.1.0 will be out, it will produce 2.1.0.gitrev which is lower than 2.1.gitrev and you won't be able to upgrade from beta to release.

rationale: NEVER use 2 numbers versions

Version string format is enforced in build/util.py

### CODE REVIEW

File: src/defs_version.h [hunk 1]
```diff
@@ -49,5 +49,5 @@
 //
 #ifndef VERSION
     // In that case, this value is used
-    #define VERSION "2.1-alpha-pre"
```
Comment (rryan) [inline src/defs_version.h]: could you add a warning about this problem in the above text? Hopefully we can avoid making this mistake again :)

Comment (sblaisot) [inline src/defs_version.h]: There was already a not above saying that windows version should be MAJOR.MINOR.patch

so I made a more visible note.

What do you think about enforcing the versionning scheme in SConscript ?

**Comment by sblaisot** [comment]
version string format is now enforced in build/util.py

**Comment by rryan** [comment]
Good idea to enforce via scons -- LGTM, thanks!

**Pull request merged**